cmd/utils/flags.go

```diff
@@ -1822,6 +1858,7 @@ func SetDNSDiscoveryDefaults(cfg *ethconfig.Config, genesis common.Hash) {
 	if url := params.KnownDNSNetwork(genesis, protocol); url != "" {
 		cfg.EthDiscoveryURLs = []string{url}
 		cfg.SnapDiscoveryURLs = cfg.EthDiscoveryURLs
+		cfg.TrustDiscoveryURLs = cfg.EthDiscoveryURLs
```

Comment:
just wonder: `TrustDiscoveryURLs` use as same configure field as `SnapDiscoveryURLs`

Reply:
yes, they use the same configure